tensorflow/compiler/xla/service/BUILD

```diff
@@ -813,11 +813,15 @@ cc_library(
     deps = [
         ":service",
         "//tensorflow/compiler/xla/service/gpu:gpu_transfer_manager",
-        "//tensorflow/compiler/xla/service/gpu:nvptx_compiler",
         "//tensorflow/core:stream_executor_no_cuda",
+        "//tensorflow/compiler/xla/service/gpu:gpu_compiler",
```

Comment:
not in this PR, but we probably need to break `gpu_compiler` into `nvptx_compiler` and `amdgpu_compiler`.